proto/raft_serverpb.proto

```diff
@@ -31,6 +41,7 @@ message RaftSnapshotData {
     optional metapb.Region region   = 1;
     optional uint64 file_size       = 2;
     repeated KeyValue data          = 3;
+    optional uint64 version         = 4;
```

Comment:
I prefer embed the meta here directly, no need to save in data with a "snapshot meta" key.